### DISCUSSION TITLE
programable IP to monitor

### DISCUSSION THREAD

**Discussion by meowserkat** [InternetConnectionMonitor/icm-public#118] [Ideas and Feature requests]
Looking to see if you add a specific IP or FQDN to monitor.  in addition to wherever it is reaching out to monitor now.   I have a specific application running in my environment and the application keeps disconnecting. But we are very certain the actual connection does not drop, but the application itself is dropping. This extension would be the perfect solution to prove the connection is still alive.

**Comment by dmishh** [comment]
@meowserkat thanks for the suggestion. Here is the same feature request requested earlier - #94 
Please vote for it if you think it would be helpful - it will give me an understanding of what to work on next
